maas2netbox/validators.py

```diff
@@ -14,7 +14,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import logging
-from urllib.parse import urlparse
+from urllib.request import urlparse
```

Comment:
This change will only work because urllib.request imports urlparse: https://github.com/python/cpython/blob/3.8/Lib/urllib/request.py#L103

Please revert it to import the urlparse function from urllib.parse which is the place where it is defined